### DISCUSSION TITLE
1.19.3及以上正版皮肤无法正常加载

### DISCUSSION THREAD

**Discussion by UmouLs** [Meloong-Git/PCL#1705] [提问] [answered]
我看之前有人反馈过这个问题 说是网络原因或者刷新皮肤就能解决
但是在我这里的情况：
官方启动器可以正常登录
HMCL正常加载皮肤
PCL2 1.19.2及以下正常加载皮肤
PCL2 1.19.3及以上被重置为默认皮肤

**Comment by MoYuan-CN** [comment]
Mojang 在 1.19.3+ 更换了皮肤算法，以至于 PCL2 的旧皮肤算法失效
1.19.3+ 皮肤自定义功能 已经于 #1288 提出，你可以到 #1596 为其投票

**Reply by UmouLs** [reply]
离线皮肤原来也会影响到正版账户的加载吗

**Comment by WForst-Breeze** [comment]
离线皮肤不会影响正版皮肤加载，一般是您与微软服务器连接出现的问题，如出现特定版本无法加载皮肤，请见下方：

如果问题稳定出现，请先检查您的网络环境，查看游戏是否安装影响皮肤加载的 mod 和资源包，并尝试在启动器中刷新您的皮肤后尝试启动。如问题仍未解决，请提交综合 bug 反馈，并附上相关启动器日志、录屏、截图和详细操作过程以辅助我们确认问题存在。十分感谢

**Comment by IF-desu** [comment, marked as answer]
先装个皮肤补丁凑合用吧，实在不行可以修改游戏jar中assets/minecraft/textures中的纹理文件(1.19.3+的皮肤有没有我还真不知道)

**Reply by UmouLs** [reply]
你说的好啊 还是皮肤补丁好使（）
真不知道他们动这个皮肤加载机制和微软认证干什么
